libs/RobotKit/include/RobotController.h

```diff
@@ -292,6 +305,8 @@ class RobotController : public interface::RobotController
 		stopActuators();
 	}
 
+	void suspendHardwareForDeepSleep() final { log_info("Entering deep sleep"); }
```

Comment:
to answer your previous comment and to make it more obvious

```suggestion
	void suspendHardwareForDeepSleep() final { 	log_info("TO IMPLEMENT - configuring hardware for deep sleep"); }

```